src-tauri/build.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
```

Comment:
All copyright comments should point to current repo:
https://github.com/mintlayer/node-gui-x/blob/master/LICENSE